nemo/collections/nlp/models/machine_translation/mt_enc_dec_model.py

```diff
@@ -435,3 +435,20 @@ def translate(self, text: List[str], source_lang: str = None, target_lang: str =
     @classmethod
     def list_available_models(cls) -> Optional[Dict[str, str]]:
         pass
+
+    def forward_for_export(self, src: torch.Tensor, src_mask: torch.Tensor):
```

Comment:
The same lines are executed in `batch_translate()` as well with the addition of detokenization. Its probably worth avoiding this redundancy. If possible, we should try and rename this function to `generate_translations()` or something like that and call it within `batch_translate()`?

Reply:
Well,  the API convention of Exportable is that if forward_for_export method is being used for ONNX export (instead of using forward()). In its current state, NMT model is not exportable as a whole anyway - this is leftover from my TS experiments - let me just remove it altogether.